ee/localserver/request-id.go

```diff
@@ -12,18 +12,25 @@ import (
 	"github.com/kolide/launcher/pkg/traces"
 )
 
-type identifiers struct {
-	UUID           string
-	InstanceId     string
-	HardwareSerial string
-}
+type (
+	identifiers struct {
+		UUID           string
+		InstanceId     string
+		HardwareSerial string
+	}
 
-type requestIdsResponse struct {
-	RequestId string
-	identifiers
-	Nonce     string
-	Timestamp time.Time
-}
+	requestIdsResponse struct {
+		RequestId string
+		identifiers
+		Nonce     string
+		Timestamp time.Time
+		Status    status
+	}
+
+	status struct {
+		EnrollmentStatus string
+	}
```

Comment:
This feels like an okay PoC, but my hunch is if this grows we'll want to centralize what our status is. Unclear if that's going to be `knapsack.Status`, `control.Status`, or `agent.Status` depends a bit on where the brain is.